### PR TITLE
Lit6370 incarnation

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
I've just found it by accident, I am afraid that it might be something problematic because https://betamasaheft.eu/works/LIT6370Incarnation/main .

### CODE REVIEW

File: new/LIT6370Incarnation.xml
Comment (thea-m) [inline new/LIT6370Incarnation.xml:9]: taśśagǝwotu ?

**Review by thea-m — CHANGES_REQUESTED** [review]

**Comment by thea-m** [comment]
I'm not sure about the error message - the record couldn't be displayed while its still unmerged. Let's wait until the merge and then check again.

File: new/LIT6370Incarnation.xml
Comment (MarcinKrawczuk) [reply new/LIT6370Incarnation.xml:9]: Leslau gives one ś

Comment (thea-m) [reply new/LIT6370Incarnation.xml:9]: I meant two g 🙈

**Review by thea-m — APPROVED** [review]

**Pull request merged**
